downloads/aws/setup-aws.sh

```diff
@@ -0,0 +1,199 @@
+#!/bin/bash
+
+check_aws_connection(){
+	print_function_details
+	CALLER_ARN=$(aws sts get-caller-identity | jq -r '.Arn')
+	if [[ -z "$CALLER_ARN" ]]; then	
+		print_error_and_exit "The aws cli does not have the propper credentials. Please fix and then try again."
+	fi
+}
+
+parse_user_input(){
+	print_function_details
+	AWS_ACCOUNT_NAME=$(jq -r '.AWS_ACCOUNT_NAME' fill-me-out.json)
+	AWS_VPC_NAME=$(jq -r '.AWS_VPC_NAME' fill-me-out.json)
+	AWS_REGION=$(jq -r '.AWS_REGION' fill-me-out.json)
+	MANAGING_ACCOUNT_ID=$(jq -r '.MANAGING_ACCOUNT_ID' fill-me-out.json)
+	AUTH_TYPE=$(jq -r '.AUTH_TYPE' fill-me-out.json)
+
+	if [[ -z "$MANAGING_ACCOUNT_ID" ]]
```

Comment:
You switch between `[[ ]]` bash checks and `[ ]` sh checks, any reason?

Reply:
No. I will fix this to be consistent.